docs/_docs/04_reference/cli-options.md

```diff
@@ -514,6 +515,20 @@ This may produce output such as:
 }
 ```
 
+### catalog
+
+Launch the user interface for searching and manipulating Terragrunt modules.
```

Comment:
NIT: Could you add a screenshot?

Reply:
I added the screenshot, but I'm not sure I did it correctly. Please check it.

Reply:
That'll do for now!